test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -536,6 +553,20 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				),
 			)).ToNot(HaveOccurred())
 
+			if mcpSelectorRemoved {
```

Comment:
but what guarantees that this will run **after** test_id:28440 , so the revert will actually happen? ginkgo by defaults run specs in random order: https://onsi.github.io/ginkgo/#spec-randomization

Reply:
@fromanirh i can try moving this code as AfterEach Section ?